frontend/tests/unit/calculators/trip-results-test.js

```diff
@@ -20,4 +19,60 @@ module('Unit | Calculator | transportation-trip-results', function (hooks) {
     // round(units/tripGenRatePerUnit) = normalizedUnits
     assert.equal(newTrCalc.normalizedUnits, 5);
   });
+
+  test('it calculates personTrips', function (assert) {
+    const units = 5;
+    const landUse = 'residential';
+    const inOutSplits = {
+      am: { in: 50, out: 50 }, md: { in: 50, out: 50 }, pm: { in: 50, out: 50 }, saturday: { in: 50, out: 50 },
+    };
+    const modes = [
+      'auto',
+      'taxi',
+      'bus',
+      'subway',
+      'railroad',
+      'walk',
+    ];
+    const manualModeSplits = false;
+    const temporalModeSplits = false;
+    const modeSplits = {
+      auto: { allPeriods: 6.8, count: 1034 },
+      taxi: { allPeriods: 6.5, count: 991 },
+      bus: { allPeriods: 0.5, count: 71 },
+      subway: { allPeriods: 57.4, count: 8771 },
+      railroad: { allPeriods: 1.9, count: 297 },
+      walk: { allPeriods: 27, count: 4129 },
+      ferry: { allPeriods: 0, count: 0 },
+      streetcar: { allPeriods: 0, count: 0 },
+      bicycle: { allPeriods: 4.4, count: 673 },
+      motorcycle: { allPeriods: 0, count: 0 },
+      other: { allPeriods: 1.4, count: 212 },
+    };
```

Comment:
This is a tiny style nit but I would just add all of this static data into the object literal that gets passed to `trCalc.create()` — the key names of that object already describe what it is and it keeps all the object input in one place. Thoughts?

Reply:
yeah, that works! It'll shorten the code slightly. I don't immediately see any extra benefits of having the static data in separate `const` variables (as it is now)

Reply:
Good idea @allthesignals and thanks @godfreyyeung - the new PR looks good!  Definitely improved the readability 👍